packages/components/slider/src/slider.ts

```diff
@@ -168,6 +168,13 @@ export const sliderProps = buildProps({
     type: Boolean,
     default: true,
   },
+  /**
+   * @description when slider inactive and `persistent` is `false` , popconfirm will be destroyed. `persistent` always be `false` when `show-tooltip ` is `false`
```

Comment:
This should be synchronized, and the table format in the above document may need to be formatted.